src/FormBuilder/resources/js/form-builder/src/components/canvas/FormSettingsContainer/components/Menu.tsx

```diff
@@ -0,0 +1,63 @@
+import {__} from '@wordpress/i18n';
+import {chevronLeft, chevronRight} from '@wordpress/icons';
+import {Icon} from '@wordpress/components';
+import {useFormSettingsContext} from '@givewp/form-builder/components/canvas/FormSettingsContainer';
+import {
+    navigateBackInMenu,
+    updateMenuState,
+} from '@givewp/form-builder/components/canvas/FormSettingsContainer/formSettingsReducer';
+import classnames from 'classnames';
+
+function MenuItem({item}) {
```

Comment:
please add `@unreleased` tag and description

Reply:
Resolved: 6e01c462287731e3f80d3f19711d3b03e30da8ce